OpenOrchestrator/Scheduler/Runner.py

```diff
@@ -92,5 +100,7 @@ def fail_job(job: Job):
 def run_process(path:str, process_name, conn_string:str, crypto_key:str):
     if path.endswith(".py"):
         return subprocess.Popen(['python', path, process_name, conn_string, crypto_key])
+    elif path.endswith(".bat"):
+        return subprocess.Popen([path, process_name, conn_string, crypto_key])
     
-    raise ValueError("The process path didn't point to a valid file: "+path)
+    raise ValueError("The process path didn't point to a valid file. Supported files are .py and .bat. Path: "+path)
```

Comment:
```suggestion
    raise ValueError(f"The process path didn't point to a valid file. Supported files are .py and .bat. Path: '{path}'")
```
In case path is a variable that cannot be concatenated to string (e.g. None), an exception would be raised